neural_lam/models/ar_model.py

```diff
@@ -328,7 +328,9 @@ def test_step(self, batch, batch_idx):
         spatial_loss = self.loss(
             prediction, target, pred_std, average_grid=False
         )  # (B, pred_steps, num_grid_nodes)
-        log_spatial_losses = spatial_loss[:, constants.VAL_STEP_LOG_ERRORS - 1]
+        log_spatial_losses = spatial_loss[
+            :, [step - 1 for step in self.args.val_steps_log]
```

Comment:
is the list comprehension here because `constants.VAL_STEP_LOG_ERRORS` was previously a numpy array but now is a list?

Reply:
yes exactly, alternatively we could convert the list from args.parse into numpy array beforehand.

Reply:
I see, not this is good :) just wanted to understand